pillar/apps/ocw-next-production.sls

```diff
@@ -8,6 +8,7 @@ ocw-next:
 
 node:
   version: 12.18.4
-  install_from_ppa: True
-  ppa:
-    repository_url: https://deb.nodesource.com/node_12.x
+  pkg:
+    use_upstream_repo: True
+    archive:
```

Comment:
From looking at the formula I don't think there's any need for the `archive` attribute. It looks like it uses the version definition to populate the appropriate repo URL

Reply:
OK, just removed it in 5d0ec698